cli/dune

```diff
@@ -25,8 +25,14 @@
  )
 )
 
+(rule
+ (targets xenops-cli.1.gz)
+ (action
+  (run gzip %{dep:xenops-cli.1}))
```

Comment:
Would gzip have to added to the opam depexts?

Reply:
Hmm... there isn't any `depext:` in the opam files of xenopsd... There must be a lot more missing. And I'm not sure about our other repos. Is this something we care to maintain?

Reply:
If it's not needed to compile the code I don't think anyone will care very much...